site/docs/v1/tech/admin-guide/_exposing-local-instance.adoc

```diff
@@ -0,0 +1,41 @@
+FusionAuth lets you develop locally with a full featured auth system. To do so, link:/docs/v1/tech/installation-guide/[install FusionAuth using one of the options, such as homebrew or a .deb file].
```

Comment:
That's a lot of text for the link. What about just putting the link on `install FusionAuth`?

```suggestion
FusionAuth lets you develop locally with a full featured auth system. To do so, link:/docs/v1/tech/installation-guide/[install FusionAuth] using one of the options, such as homebrew or a .deb file.
```